src/main/java/Decide.java

```diff
@@ -47,6 +47,49 @@ public boolean LIC0() throws IllegalParameterObjectException {
         return false;
     }
 
+    /**
+     * Check if LIC2 is true.
+     * Calculates the angle between three consecutive points, the second point being located at the vertex
+     * of the angle.
+     *
+     * @return true if the angle < pi - epsilon or angle > pi + epsilon
+     *
+     */
+    public boolean LIC2() throws IllegalParameterObjectException {
+        if (parameters.getEPSILON() < 0 || parameters.getEPSILON() > Math.PI) {
```

Comment:
The function looks good! The only thing is it's supposed to be >= Math.PI. Change that and I can merge it.